AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -2164,6 +2164,7 @@ hipertextual.com##.mrf-newsletter
 astro-world.ru##.theiaStickySidebar > aside.penci_popular_news_widget + aside.widget
 ria.ru##.article__aggr
 wisn.com##iframe[src^="https://d2cmvbq7sxx33j.cloudfront.net/email/prod_coronavirus_iframe_section.html"]
+||drozdogan.com/images/koronavirus_banner_big.gif
```

Comment:
`||drozdogan.com/images/koronavirus_banner_*.gif`

Reply:
and why in newsletter section?

Reply:
@Sergey-Lyapin, not quite sure where to put it.